src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,30 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import java.io.IOException; // OK
+ * import java.net.URL; // OK
+ *
+ * import javax.net.ssl.SSLContext; // OK
+ * import javax.net.ssl.TrustManager; // OK
+ *
+ * import org.apache.http.conn.ClientConnectionManager; // OK
+ * import org.apache.http.conn.scheme.Scheme; // OK
+ *
+ * import com.neurologic.http.HttpClient; // OK
+ * import com.neurologic.http.impl.ApacheHttpClient; // OK
+ *
+ * public class SomeClass { ... }
```

Comment:
See my comment at https://github.com/checkstyle/checkstyle/pull/9165#issuecomment-757324280. Please make sure that all such examples follow this.

Reply:
okay

Reply:
i have checked rest of the example
Pls check [This](https://github.com/checkstyle/checkstyle/pull/9165#issuecomment-757420779)
I am getting this type of error everywhere if we say to ignore this error then i think we are good to go .

Reply:
Resolved